.prettierignore

```diff
@@ -0,0 +1,11 @@
+LICENSE.md
+assets
+build
+main
+node_modules
+renderer
+test
+*.js
+*.json
+**/*.yml
+**/*.yaml
```

Comment:
Why exclude these files?

Reply:
We are using [standard](https://www.npmjs.com/package/standard) coding style. It is unfortunately incompatible with Prettier; see https://github.com/standard/standard/issues/996 and the linked issues.

My goal here is to configure Prettier to lint only Markdown files.

I am ok with adopting Prettier for Yaml & JSON files if you like?

Please note that `npm` may produce `package.json` in a different format than Prettier, so we should probably keep that one excluded.

Reply:
```suggestion
*
!**.md
```

> My goal here is to configure Prettier to lint only Markdown files.

I think this expresses the intend more clearly (assuming it works).

> I am ok with adopting Prettier for Yaml & JSON files if you like?

I think standard covers these as well, no need from me.

Reply:
Great suggestion!

I reworked the prettier ignore file based on your suggestion and also added comments to explain what's going on.

See 30eea22

@juliangruber LGTY now?